MmSupervisorPkg/Core/Misc/MemoryAttributesTable.c

```diff
@@ -352,6 +352,75 @@ SmmCreateImageRecordInternal (
   return Status;
 }
 
+EFI_STATUS
+EFIAPI
+ProtectReadonlyData (
+  IN EFI_MM_DRIVER_ENTRY  *DriverEntry,
+  IN BOOLEAN              IsSupervisorImage
+  )
+{
+  UINTN                                Index;
+  EFI_IMAGE_DOS_HEADER                 *DosHdr;
+  UINT32                               PeCoffHeaderOffset;
+  EFI_IMAGE_OPTIONAL_HEADER_PTR_UNION  Hdr;
+  UINT32                               SectionAlignment;
+  EFI_IMAGE_SECTION_HEADER             *Section;
+  UINT64                               SectionStart;
+  UINT64                               SectionEnd;
+  EFI_PHYSICAL_ADDRESS                 ImageBase;
+
+  if (DriverEntry == NULL) {
+    ASSERT (FALSE);

```

Comment:
Would print the assertion condition more clearly as:

```suggestion
    ASSERT (DriverEntry != FALSE);
```